### ISSUE TITLE
kind=8 error with gfortran 7.2

### ISSUE THREAD

**Issue by rhiannonlynne** [oorb/oorb#37]
When compiled with gfortran (7.2.0), line 90 in cl_options.f90 compiles without errors, but gives an error when called with any orbit fitting task. 

example of failure:
```
[ewok:~/lsstRepos/oorb/modules] lynnej% oorb --task=ranging --obs-in=$HOME/mpc.mpc --orb-out=orb.des
At line 68 of file cl_options.f90
Internal Error: get_unit(): Bad internal unit KIND

Error termination. Backtrace:
#0  0x1068d058c
#1  0x1068d1295
#2  0x1068d174d
#3  0x106a14f26
#4  0x106a1281f
#5  0x1068196d0
#6  0x10667371c
#7  0x10686810e
```

**Comment by rhiannonlynne** [comment]
That said, I tried out KindFinder and I guess I just don't know enough about fortran to know if this means anything? 

```
             The Numerical Models of
          Integer & Real/Complex Types
  
      available with this Computer/Compiler
  
  
  Author: Werner W Schulz, (C) 1998
  
  This Programme was generated automatically by KindFinder,
  Author: Werner W Schulz, (C) 1998
  The Names used below are those used in KindFinder
  


 INTEGER MODEL:
  Name:                     Int_01              Int_02             Int_Def              Int_08              Int_16
  KIND:                          1                   2                   4                   8                  16
  DIGITS:                        7                  15                  31                  63                 127
  RADIX:                         2                   2                   2                   2                   2
  RANGE:                         2                   4                   9                  18                  38
  HUGE:                        127               32767          2147483647 9223372036854775807********************


 FLOATINGPOINT MODEL (Real/Complex):
  Name:                     Single              Double              Extnd1              Extnd2
  KIND:                          4                   8                  10                  16
  DIGITS:                       24                  53                  64                 113
  RADIX:                         2                   2                   2                   2
  MINEXPONENT:                -125               -1021              -16381              -16381
  MAXEXPONENT:                 128                1024               16384               16384
  PRECISION:                     6                  15                  18                  33
  RANGE:                        37                 307                4931                4931
  EPSILON:               1.192E-07           2.220E-16           1.084E-19           1.926E-34
  HUGE:                  3.403E+38           1.798+308****************************************
  TINY:                  1.175E-38           2.225-308****************************************
```

**Comment by rhiannonlynne** [comment]
Interestingly, this behavior seems to go away if I compile with the 'debug' options (./configure gfortran deb) instead of opt.

although this was the result:  (still didn't get an orbit -- this may be my fault for some other misconfiguration??)
```[ewok:~/lsstRepos/oorb/main] lynnej% ./oorb --task=ranging --obs-in=$HOME/mpc.mpc --orb-out=out.des
  WARNING: NOT using dchi2 in acceptance!
  WARNING: REGULARIZATION is OFF!
***ERROR*** 26 Oct 2017 00:22:45UTC (StochasticOrbit / statisticalRanging) No sample orbits found!
***ERROR*** 26 Oct 2017 00:22:45UTC (StochasticOrbit / autoStatisticalRanging) First iteration failed.
***ERROR*** 26 Oct 2017 00:22:45UTC (oorb / ranging) Ranging failed:
ID:     A and number of observations:  41
```
 -- this is just using the current MPC observations from http://www.minorplanetcenter.net/mpec/K17/K17UI3.html

**Comment by mgranvik** [comment]
Thanks for reporting this bug. I've fixed the problem with pull request #38 . Let me know if it solves the problem.

Mikael
